apps/silverback-gatsby/src/templates/article.tsx

```diff
@@ -21,11 +22,9 @@ export const query = graphql`
         title
       }
       image {
-        translation(langcode: $langcode) {
-          alt
-          localImage {
-            ...ImageSharpFixed
-          }
+        alt
+        localImage {
+          ...ImageSharpFixed
```

Comment:
Do we still get the `alt` text in the correct language?

Reply:
I'm not sure. Maybe. But if this is the case, it should be fixed in https://github.com/drupal-graphql/graphql/pull/974

Vasi already uses this patch on his project.

Reply:
Actually, even if it is fixed in the upstream, there still can be issues. E.g.
- a translatable entity references an untranslatable one, which in turn references a translatable entity
  - the language will be lost on the interim entity
- an untranslatable page references a translatable entity
  - here we don't know the current locale in the page template

I'd say, let's just see how it goes 😅 In most cases all entities are translatable and it's a rare case when we have more than one reference level.

Reply:
Its not just about entity references. What about entities that are embedded in gutenberg documents?

Reply:
I think `load[Type]` should set the current language context and all sub-fields should respect that when loading other entities. But that is not in scope of this PR.